src/EditorFeatures/Core/InlineRename/InlineRenameUIOptions.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable disable
```

Comment:
I'm merging the Cocoa/WPF here. @sandyarmstrong do we need `#nullable disable` here?